source/exec.tex

```diff
@@ -5672,3 +5687,424 @@
 return as_awaitable(std::forward<Value>(value), static_cast<Promise&>(*this));
 \end{codeblock}
 \end{itemdescr}
+
+\rSec1[exec.par.scheduler]{Parallel scheduler}
+
+\pnum
+\tcode{parallel_scheduler} models \libconcept{scheduler}.
+
+\pnum
+Let \tcode{sch} be an object of type \tcode{parallel_scheduler}, and
+let \tcode{\exposid{BACKEND-OF}(sch)} be \tcode{*ptr},
+where \tcode{sch} is associated with \tcode{ptr}.
+
+\pnum
+The expression \tcode{get_forward_progress_guarantee(sch)} returns
+\tcode{forward_progress_guarantee::paral\-lel}.
+
+\pnum
+Let \tcode{sch2} be an object of type \tcode{parallel_scheduler}.
+Two objects \tcode{sch} and \tcode{sch2} compare equal if and only if
+\tcode{\exposid{BACKEND-OF(sch)}} and
+\tcode{\exposid{BACKEND-OF(sch2)}} refer to the same object.
+
+\pnum
+Let \tcode{rcvr} be a receiver.
+A \defn{proxy} for \tcode{rcvr} with base \tcode{B} is
```

Comment:
The paper says

> A *proxy for `rcvr` with base `B`* is an lvalue ...

So presumably, what's being defined is "proxy for with base", not just the term "proxy".

Reply:
I agree :-)